packages/service-library/src/servicelib/aiohttp/tracing.py

```diff
@@ -91,6 +99,10 @@ def setup_tracing(
 
     # Instrument aiohttp client
     AioHttpClientInstrumentor().instrument()
-    if instrument_aiopg:
+    if HAS_AIOPG:
         AiopgInstrumentor().instrument()
+        _logger.info("Attempting to add aiopg opentelemetry autoinstrumentation...")
```

Comment:
Q: it's done by this point no?

```suggestion
        _logger.info("Added aiopg opentelemetry instrumentation.")
```